### PR TITLE
runstatus: Add explanation for custom domains

### PR DESCRIPTION
**Pull request by TooAngel**

### CODE REVIEW

**Comment by pst** [comment]
LGTM

File: exoscale_documentation/runstatus/00_runstatus-quick-start.md [hunk 1]
```diff
@@ -153,4 +155,12 @@ You'll receive a response similar to this:
...
-You can post updates to your status page using the Runstatus API. Soon we'll publish a new guide specifically about using that API.
\ No newline at end of file
+You can post updates to your status page using the Runstatus API. Soon we'll
+publish a new guide specifically about using that API.
+
+## Enabling a custom domain
+
+To use your own domain instead of `NAME.runstat.us` you can simply enter the
+domain in the `General Settings` of you status page. Additionally your DNS
+server needs to be configured to point you status domain to `NAME.runstat.us`
```
Comment (brutasse) [inline exoscale_documentation/runstatus/00_runstatus-quick-start.md]: your*

File: exoscale_documentation/runstatus/00_runstatus-quick-start.md [hunk 2]
```diff
@@ -19,6 +19,7 @@ In this quick-start, you'll learn how to:
 	* post a service status update
 	* schedule and announce a maintenance window
  * use finger to query the status of an application from your terminal.
+ * use a custom domain
```
Comment (brutasse) [inline exoscale_documentation/runstatus/00_runstatus-quick-start.md]: Trailing dot from line above should be moved to this line (same L45)

File: exoscale_documentation/runstatus/00_runstatus-quick-start.md (hunk 1 above)
Comment (TooAngel) [inline exoscale_documentation/runstatus/00_runstatus-quick-start.md]: thx

File: exoscale_documentation/runstatus/00_runstatus-quick-start.md (hunk 2 above)
Comment (TooAngel) [inline exoscale_documentation/runstatus/00_runstatus-quick-start.md]: thx, will fix and push to master

**Pull request merged**